<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EXFO_OperationalTest_T100_SWT_ATN_Window</class>
 <widget class="QMainWindow" name="EXFO_OperationalTest_T100_SWT_ATN_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>786</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="runTestButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>620</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="outuptFilenameLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>711</width>
      <height>41</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="outputFileButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>530</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Choose Output File</string>
    </property>
   </widget>
   <widget class="QLabel" name="directionsTextField">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>341</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PLACEHOLDER TEXT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="directionsImageLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>551</width>
      <height>461</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/EXFO_DRAWING_OPTEST1_SETUP_T100_POWER_STEP_TEST.PNG</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>runTestButton</zorder>
   <zorder>outuptFilenameLineEdit</zorder>
   <zorder>outputFileButton</zorder>
   <zorder>directionsImageLabel</zorder>
   <zorder>directionsTextField</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
